x-pack/plugins/security_solution/public/cases/components/connectors/case/existing_case.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useMemo, useCallback } from 'react';
+import { useGetCases } from '../../../containers/use_get_cases';
+import { useCreateCaseModal } from '../../use_create_case_modal';
+import { CasesDropdown } from './cases_dropdown';
+
+interface ExistingCaseProps {
+  selectedCase: string | null;
+  onCaseChanged: (id: string) => void;
+}
+
+const ExistingCaseComponent: React.FC<ExistingCaseProps> = ({ onCaseChanged, selectedCase }) => {
+  const { data: cases, loading: isLoadingCases, refetchCases } = useGetCases();
+
+  const onCaseCreated = useCallback(() => refetchCases(), [refetchCases]);
+
+  const { Modal: CreateCaseModal, openModal } = useCreateCaseModal({ onCaseCreated });
+
+  const onChange = useCallback(
+    (id: string) => {
+      if (id === 'add-case') {
```

Comment:
let's use a enum here

Reply:
There is no other value than `add-case`. This is the id of the value of the last option of the dropdown selection. It is the `Add new case` button.

Reply:
I think that it is safer so nobody can change it by mistake or can we just create a const like `const ADD_CASE = 'add-case'` and if you have a unit test will should be good to go.